infra-provider-kafka/src/main/java/com/homeaway/streamplatform/streamregistry/provider/impl/KafkaInfraManager.java

```diff
@@ -254,8 +254,12 @@ void updateTopics(ZkUtils zkUtils, List<String> topicsToUpdate, Map<String, Stri
             // to exactly match downstream config when the stream-registry has not "onboarded" existing topic
             // for the first time.
 
-            // TODO Alternatively we can add a forceSync=true flag, ignoring any user provided info, and only updating SR with the underlying settings
-            //      We should probably do forceSync=true anyway, as it provides a simple way to keep things in sync (#114)
+            // Added a forceSync=true flag, ignoring any user provided info, and only updating SR with the underlying settings (#114)
+            if (forceSync) {
+                // NOTHING TO DO!
+                log.info("topic config mismatch for {} ignored. Input config={}, actual (retained) config={}", topic, topicConfigMap, actualTopicConfig);
+                continue;
```

Comment:
What exactly is being "forced" if all you do is `continue`?

Reply:
Well, the flag is "forcing" the stream-registry to sync the given topic configs with what exists already on the underlying brokers.
I guess I can say that by `continue`ing here, the code is "forcing" the flow to not go ahead to the following exception generation ..

Reply:
I put alternative names in #114

Reply:
> ignoring any user provided info, and only updating SR with the underlying settings

The way I interpret that is to take `actualTopicConfig` and overwrite `topicConfigMap` with it...

Reply:
Doesn't `continue` actually perform the UPDATE???
I agree with @cricket007 remarks on `actualTopicConfig` and `topicConfigMap`... where we should be overriding what the user provided, with what is _actually_ out there.

Reply:
> Doesn't continue actually perform the UPDATE?

No, it breaks here and goes to the next iteration of the loop. `updateTopic` isn't called.

Reply:
Oh. I missed a curly-brace close.

line 270 is INSIDE not... outside the loop.
https://github.com/ExpediaGroup/stream-registry/pull/170/files/09090b33c68b43bf046a36ddd7b7ed2959b849e3#diff-b4dcb632004addc439896ec21e5b7dabR270